test/eventbrite-provider.test.ts

```diff
@@ -35,6 +47,20 @@ describe('eventbrite-provider', () => {
     await seneca.ready()
   })
 
+  test('entity-load', async () => {
+    const seneca = Seneca({ legacy:false })
+      .test()
+      .use('promisify')
+      .use('entity')
+      .use('provider', providerOptions)
+      .use(EventbriteProvider)
+      
+    const event = await seneca.entity('eventbrite/event').load$('214728557897');
```

Comment:
Is this the id of a public event that you created?
Does the event have public URL?

Reply:
It's a id a took from an public event in Ireland. Is it correct to take a ID from a random event ? should I create a new one ?
https://www.eventbrite.com/e/meet-santa-in-castlebar-tickets-214728557897?aff=ebdssbcitybrowse&keep_tld=1